src/components/QuizzQuestion.js

```diff
@@ -65,75 +119,155 @@ const QuizzQuestion = ({
+      <Grid
+        container
+        justify="center"
+        alignItems="center"
+        className={classes.module}
+        spacing={2}
+      >
 
-      <Typography variant="h1" gutterBottom>
-        {t('Quizz question page')}
-      </Typography>
+        <Typography className={classes.module__title} variant="h1" gutterBottom>
+          {t('Farm vulnerability and adaptation Quiz')}
+        </Typography>
 
-      <Typography variant="h3" component="h2">
-        {t('Question')}
-      </Typography>
+        <Typography className={classes.category__title} variant="h2" gutterBottom>
+          {t('Question')}
+        </Typography>
 
-      <Typography
-        variant="body1"
-        dangerouslySetInnerHTML={{ __html: question }}
-      />
+        <Typography
+          variant="subtitle1"
+          dangerouslySetInnerHTML={{ __html: question }}
+        />
 
-      <Typography variant="h4" component="h3">
-        {t('Proposals')}
-      </Typography>
+        <Grid
+          item
+          xs={12}
+          md={9}
+          xl={8}
 
-      {answers.map(({ text, valid }, index) => (
-        <div
-          key={text}
           className={clsx({
-            [classes.answer]: true,
-            [classes.valid]: isAnswered && valid,
-            [classes.invalid]: isAnswered && !valid,
-            [classes.userChoice]: givenAnswer === index,
+            [classes.answers__group]: true,
           })}
+
+        >
+          {answers.map(({ text, valid }, index) => (
+            <Grid
+              container
+              direction="column"
+              justify="space-aroundflex-start"
+              alignItems="stretch"
+              key={text}
+              className={clsx({
+                [classes.answer]: true,
+                [classes.valid]: isAnswered && valid,
+                [classes.invalid]: isAnswered && !valid,
+                [classes.validUserChoice]: givenAnswer === index && valid,
+                [classes.invalidUserChoice]: givenAnswer === index && !valid,
+              })}
+            >
+              <Button
+                variant="outlined"
+                onClick={() => dispatch({
+                  type: 'ANSWER',
+                  payload: { id, theme, index, valid },
+                })}
+              >
+                {text}
+              </Button>
+            </Grid>
+          ))}
+        </Grid>
+        <Grid
+          container
+          direction="column"
+          justify="center"
+          alignItems="center"
+        >
+          <Grid item xs={12} md={9} xl={8}>
+            <Card className={classes.card} elevation={6}>
+              <CardContent>
+                <Typography variant="h3" component="h2">
+                  {t('Explanation')}
+                </Typography>
+
+                <Typography
+                  variant="body2"
+                  dangerouslySetInnerHTML={{ __html: explanation.slice(350) }}
```

Comment:
Do we really need to limit text length?

Also, you can't slice like this as you me cut inside an html tag...

Reply:
it was for testing, done